src/js/components/Footer/stories/Simple.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Footer, grommet, Grommet, Main, Text } from 'grommet';
+
+const Simple = () => (
+  <Grommet theme={grommet}>
+    <Main
+      background="light-1"
+      elevation="large"
+      pad={{ top: 'large' }}
+      gap="large"
+    >
+      <Text margin="small" size="xsmall">
+        Main Content
+      </Text>
+      <Box flex />
+      <Footer background="light-4" justify="center">
+        <Box pad="small">
```

Comment:
why not put the pad on the Footer?

Reply:
Because essentially it works the same and I thought it will be better to assign it only for the relevant component?!
If you are asking because you potentially wants to remove the Box, the Bos is needed to default to `direction="column"`.